src/pages/staticPages/version/Version.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { PageLayout } from '../../../shared/components/PageLayout/PageLayout';
+import './styles.scss';
+import AppConfig from 'appConfig';
+import StaticContent from '../../../shared/components/staticContent/StaticContent';
+import Helmet from 'react-helmet';
+
+export default class Version extends React.Component<{}, {}> {
+    public render() {
+        return (
+            <PageLayout className="whiteBackground staticPage">
+                <Helmet>
+                    <title>{'cBioPortal for Cancer Genomics::Version'}</title>
```

Comment:
Maybe `cBioPortal for Cancer Genomics Version`? The `::` looks a bit too technical

Reply:
Sure, will change that!